superset/views/annotations.py

```diff
@@ -46,6 +46,9 @@ class AnnotationModelView(
     SupersetModelView, DeleteMixin
 ):  # pylint: disable=too-many-ancestors
     datamodel = SQLAInterface(Annotation)
+
+    class_permission_name = "Annotation"
```

Comment:
Not sure about this Output either

![Uploading Screenshot from 2020-04-03 04-30-45.png…]()